cmake/partition_manager.cmake

```diff
@@ -19,6 +19,9 @@ macro(add_region)
     list(APPEND region_arguments
       "--${REGION_NAME}-dynamic-partition;${REGION_DYNAMIC_PARTITION}")
   endif()
+
+  message("Added region with size ${REGION_SIZE}")
+  message("Added region with base  ${REGION_BASE}")
```

Comment:
These printouts could have more context, like
```suggestion
  message("Partition Manager: Added region with size ${REGION_SIZE}")
  message("Partition Manager: Added region with base  ${REGION_BASE}")
```

Reply:
I removed these, unsure why I added them in the first place.